examples/image_reconstruction.ipynb

```diff
@@ -0,0 +1,705 @@
+      "text/plain": [
+       "<Figure size 720x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light",
+      "tags": []
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Calculating predictions of the trained network on a batch\n",
+    "outputs = torch.tensor(model.predict_on_batch(inputs))\n",
+    "\n",
+    "output_grid = make_grid(outputs)\n",
+    "fig = plt.figure(figsize=(10, 10))\n",
+    "out = output_grid.numpy().transpose((1, 2, 0))\n",
+    "plt.imshow(out)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "e4kdBqMsw_P5"
+   },
+   "source": [
+    "The code below show how we can takte advantages of poutyne to resume training from a special epoch (for example if we found the past epoch not enough, we can resume from the last done epoch, as below:"
```

Comment:
This needs rewording since the second part repeats the first one.

Reply:
I like the idea of just restarting the training, this should the advantage of having ckpt. Strong like.